### PR TITLE
Icon meaning

### PR DESCRIPTION
**Pull request by RomiKMoveo**
Implemented IconsMeaning component that will be presented in the HomePage.

Exported icons from Figma to assert/icons.
Added component style and implementation. 

none

I have:
- [ ] deleted all irrelevant comments, logs and debuggers.
- [ ] assure that I don't print full error stacks, full body response or headers from the server in the console.
- [ ] deleted all sensitive information from the code because it's being imported from environment variables.
- [ ] encrypted keys/passwords that are sent through the internet.
- [ ] sanitized all user inputs to prevent attacks.
- [ ] made sure my code is generic and reusable for future things or reused existing resources.
- [ ] made sure that my code is readable.
- [ ] checked that the functions and resource names are self-explanatory.
- [ ] tested the code locally and it works as expected.
- [ ] added validations and error handling in relevant cases (external requests, integrations, etc).

### CODE REVIEW

File: epicure-frontend/src/pages/home/components/IconsMeaning/IconsMeaning.tsx [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
     <>
-      <div>IconsMeaning</div>
+      <section className="icons-meaning-container">
+        <div className="icons-meaning-title ">THE MEANING OF OUR ICONS:</div>
+        <ul className="icons-list-container">
+            {iconsMeaningIcons.map(({ icon, alt, caption }) => (
+              <figure key={alt} className="icon-meaning-item">
+                <img src={icon} alt={alt} className="icon-img" />
+                <figcaption className="icon-caption">{caption}</figcaption>
+              </figure>
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/pages/home/components/IconsMeaning/IconsMeaning.tsx]: set it as a separate component

File: epicure-frontend/src/pages/home/components/Footer/Footer.scss [hunk 2]
```diff
@@ -1,16 +1,61 @@
...
+footer {
     display: flex;
-    //flex-direction: column;
-    padding-left: 20px;
-    padding-right: 20px;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    height: 202px;
+    top: 3763px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/pages/home/components/Footer/Footer.scss:12]: why needed the top if ur using flex?

File: epicure-frontend/src/pages/home/components/Footer/Footer.scss [hunk 3]
```diff
@@ -1,16 +1,61 @@
...
+
+footer {
     display: flex;
-    //flex-direction: column;
-    padding-left: 20px;
-    padding-right: 20px;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    height: 202px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/pages/home/components/Footer/Footer.scss:11]: see if needed fixed size

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]

File: epicure-frontend/src/pages/home/components/Footer/Footer.scss (hunk 2 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/pages/home/components/Footer/Footer.scss:12]: dealt with in the former pr

File: epicure-frontend/src/pages/home/components/IconsMeaning/IconsMeaning.tsx (hunk 1 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/pages/home/components/IconsMeaning/IconsMeaning.tsx]: done

File: epicure-frontend/src/pages/home/components/Footer/Footer.scss (hunk 3 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/pages/home/components/Footer/Footer.scss:11]: dealt with in the former pr

**Pull request merged**